src/components/templates/ExternalExperience/ExternalExperience.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect } from "react";
+import { useCss } from "react-use";
+import cn from "classnames";
+
+import { STRING_SPACE } from "settings";
+
+import { AnyVenue } from "types/venues";
+
+import { WithId } from "utils/id";
+import { externalUrlAdditionalProps, openUrl } from "utils/url";
+
+import { RenderMarkdown } from "components/organisms/RenderMarkdown";
+
+import CN from "./ExternalExperience.module.scss";
+
+const componentRules = {
```

Comment:
I think think this one can also be put in `Object.freeze()` not as much as expecting the `RenderMarkdown` to change it, but more as a signal that the object is a constant.

Reply:
done: https://github.com/sparkletown/sparkle/pull/3015/commits/42838ddbe607e71b4b4b587e9c83c1883992155b